web/frontend/styles/casebook.scss

```diff
@@ -167,10 +167,26 @@ main > header.casebook {
       border: none;
     }
 
-    &.export { background-image: url('~static/images/ui/casebook/export.svg'); }
-    &.export.animating { background-image: url('~static/images/ui/casebook/export-anim.svg'); }
-    &.export.fancy { background-image: url('~static/images/ui/casebook/export-fancy.svg'); }
-    &.export.fancy.animating { background-image: url('~static/images/ui/casebook/export-anim-fancy.svg'); }
+    &.export {
+        background-image: url('~static/images/ui/casebook/export.svg');
+        &::after{
+            position:absolute; width:0; height:0; overflow:hidden; z-index:-1;
+            content: url('~static/images/ui/casebook/export-anim.svg');
```

Comment:
Preload the animated version of the export animation in an invisible pseudo element so that Safari doesn't make an aborted later attempt to load it.